src/networkaccessmanager.cpp

```diff
@@ -2,7 +2,7 @@
   This file is part of the PhantomJS project from Ofi Labs.
 
   Copyright (C) 2011 Ariya Hidayat <ariya.hidayat@gmail.com>
-  Copyright (C) 2011 Ivan De Marino <ivan.de.marino@gmail.com>
+  Copyright (C) 2011 Ivan De Marino <ivan.de.marino@gmail.com>s
```

Comment:
Great work. Could definitely use some better reporting. I assume the added 's' here is errant.